packages/http/configCache.js

```diff
@@ -1,4 +0,0 @@
-angular.module('wfm.http').config(['$httpProvider',
-  function($httpProvider) {
-    $httpProvider.defaults.cache = true;
-  }]);
```

Comment:
:+1: for removing this, can be configured in the app level if required.

The caching seems to be too aggressive, hard to pick up updates. express already handles 304 responses nicely, so no need to for aggressive client caching in the portal.